source/Body.h

```diff
@@ -94,10 +96,10 @@ class Body {
 	Point position;
 	Point velocity;
 	Angle angle;
+	Point scale;
```

Comment:
```suggestion
	Point scale = Point(1, 1);
```
not setting it here changes from previous behavior, is that intended?